civicaseextras.php

```diff
@@ -0,0 +1,163 @@
+  _civicaseextras_civix_civicrm_xmlMenu($files);
+}
+
+/**
+ * Implements hook_civicrm_install().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_install
+ */
+function civicaseextras_civicrm_install() {
+  _civicaseextras_civix_civicrm_install();
+}
+
+/**
+ * Implements hook_civicrm_postInstall().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_postInstall
+ */
+function civicaseextras_civicrm_postInstall() {
+  _civicaseextras_civix_civicrm_postInstall();
+}
+
+/**
+ * Implements hook_civicrm_uninstall().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_uninstall
+ */
+function civicaseextras_civicrm_uninstall() {
+  _civicaseextras_civix_civicrm_uninstall();
+}
+
+/**
+ * Implements hook_civicrm_enable().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_enable
+ */
+function civicaseextras_civicrm_enable() {
+  _civicaseextras_civix_civicrm_enable();
+}
+
+/**
+ * Implements hook_civicrm_disable().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_disable
+ */
+function civicaseextras_civicrm_disable() {
+  _civicaseextras_civix_civicrm_disable();
+}
+
+/**
+ * Implements hook_civicrm_upgrade().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_upgrade
+ */
+function civicaseextras_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
+  return _civicaseextras_civix_civicrm_upgrade($op, $queue);
+}
+
+/**
+ * Implements hook_civicrm_managed().
+ *
+ * Generate a list of entities to create/deactivate/delete when this module
+ * is installed, disabled, uninstalled.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_managed
+ */
+function civicaseextras_civicrm_managed(&$entities) {
+  _civicaseextras_civix_civicrm_managed($entities);
+}
+
+/**
+ * Implements hook_civicrm_caseTypes().
+ *
+ * Generate a list of case-types.
+ *
+ * Note: This hook only runs in CiviCRM 4.4+.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_caseTypes
+ */
+function civicaseextras_civicrm_caseTypes(&$caseTypes) {
+  _civicaseextras_civix_civicrm_caseTypes($caseTypes);
+}
+
+/**
+ * Implements hook_civicrm_angularModules().
+ *
+ * Generate a list of Angular modules.
+ *
+ * Note: This hook only runs in CiviCRM 4.5+. It may
+ * use features only available in v4.6+.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_angularModules
+ */
+function civicaseextras_civicrm_angularModules(&$angularModules) {
+  _civicaseextras_civix_civicrm_angularModules($angularModules);
+}
+
+/**
+ * Implements hook_civicrm_alterSettingsFolders().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_alterSettingsFolders
+ */
+function civicaseextras_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
+  _civicaseextras_civix_civicrm_alterSettingsFolders($metaDataFolders);
```

Comment:
Not defined yet?

Reply:
Also defined in `civicaseextras.civix.php`:

https://github.com/compucorp/uk.co.compucorp.civicaseexrtas/pull/1/files#diff-3e362daca7fbcc9c78d00b63e88ef045R463

This was defined by the plugin by default, and no modification was made.